apps/hubble-stats/containers/PoolOverviewContainer/PoolOverviewContainer.tsx

```diff
@@ -29,72 +29,75 @@ export default async function PoolOverviewContainer({
   return (
     <div
       className={cx(
-        'w-full space-y-4 p-6 rounded-lg',
+        'w-full lg:h-[278px] p-6 rounded-lg',
```

Comment:
The hardcoded height used here is not ideal for responsiveness. Could you please consider utilizing a different property to manage the height, such as `min-height` or `max-height`?